engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -472,7 +472,7 @@ namespace dmGameSystem
                     }
                     else if (info->m_Attribute->m_CoordinateSpace == dmGraphics::COORDINATE_SPACE_LOCAL)
                     {
-                        memcpy(vertices_write_ptr, &p, info->m_ValueByteSize);
+                        memcpy(vertices_write_ptr, &p_local, info->m_ValueByteSize);
```

Comment:
Is the `m_ValueByteSize` guarantueed to be the correct size for the fixed semantic types?

Reply:
Good question. It might not be if you specify the position stream as a bigger type than what we supply. What will happen is that you are going to get junk values since we are copying outside of the P3 range, but not a crash.